<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="1" failed="0" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-07-29T13:45:51 IST" name="Default suite" finished-at="2024-07-29T13:45:51 IST" duration-ms="51">
    <groups>
    </groups>
    <test started-at="2024-07-29T13:45:51 IST" name="Default test" finished-at="2024-07-29T13:45:51 IST" duration-ms="51">
      <class name="CoreJava.clsExceptionHandling">
        <test-method signature="ArrayIndexException()[pri:0, instance:CoreJava.clsExceptionHandling@7de62196]" started-at="2024-07-29T13:45:51 IST" name="ArrayIndexException" finished-at="2024-07-29T13:45:51 IST" duration-ms="7" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ArrayIndexException -->
      </class> <!-- CoreJava.clsExceptionHandling -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
